src/components/CytoscapeGraph/CytoscapeToolbar.tsx

```diff
@@ -28,16 +35,52 @@ export class CytoscapeToolbar extends React.PureComponent<CytoscapeToolbarProps>
         <Button onClick={this.zoomIn}>
           <Icon type="fa" name="plus" />
         </Button>
-        <Button onClick={this.zoomOut} className={cytoscapeToolbarButtonStyle}>
+
+        <Button onClick={this.zoomOut}>
           <Icon type="fa" name="minus" />
         </Button>
-        <Button onClick={this.fit} className={cytoscapeToolbarButtonStyle}>
+
+        <Button onClick={this.fit} className={cytoscapeToolbarPadWideStyle}>
           <div className="glyphicon glyphicon-fullscreen" />
         </Button>
+
+        <Button
+          onClick={() => {
+            this.props.onLayoutChange(DagreGraph.getLayout());
+          }}
+          className={cytoscapeToolbarPadWideStyle}
+          title={DagreGraph.getLayout().name}
+          active={this.props.activeLayout.name === DagreGraph.getLayout().name}
+        >
+          <div className="fa pficon-infrastructure fa-rotate-270" />
+        </Button>
+
+        <Button
+          onClick={() => {
+            this.props.onLayoutChange(CoseGraph.getLayout());
+          }}
+          className={cytoscapeToolbarPadThinStyle}
+          title={CoseGraph.getLayout().name}
+          active={this.props.activeLayout.name === CoseGraph.getLayout().name}
+        >
+          <div className="fa pficon-topology" /> 1
+        </Button>
+
+        <Button
+          onClick={() => {
+            this.props.onLayoutChange(ColaGraph.getLayout());
+          }}
+          className={cytoscapeToolbarPadThinStyle}
+          title={ColaGraph.getLayout().name}
+          active={this.props.activeLayout.name === ColaGraph.getLayout().name}
+        >
+          <div className="fa pficon-topology" /> 2
+        </Button>
```

Comment:
You probably want to wrap these Buttons in a [ButtonGroup](https://rawgit.com/patternfly/patternfly-react/gh-pages/index.html?knob-Style=default&selectedKind=patternfly-react%2FWidgets%2FButton&selectedStory=ButtonGroup&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs)